### PR TITLE
reorder and update releases usage pages

### PR DESCRIPTION
**Pull request by imatwawana**
- Reorders pages under Usage
- Retitles Change Time Range page
- General style/wording fixes

### CODE REVIEW

File: src/docs/product/releases/usage/sorting-filtering.mdx [hunk 1]
```diff
@@ -1,27 +1,29 @@
...
 - [Active Sessions/Users](/product/releases/health/#active-sessionsusers)
 - [Crash Free Sessions/Users](/product/releases/health/#crash-free-sessionsusers)
 - Build Number
 - [Semantic Version](/platform-redirect/?next=/configuration/releases/%23bind-the-version)
+- Date Adopted (when you select a single environment)
 
 ## Filtering Releases
 
-Search on the **Releases** page supports both raw text and query syntax. 
+Search on the **Releases** page supports both raw text and query syntax and you can search using the following properties:
```
Comment (imatwawana) [inline src/docs/product/releases/usage/sorting-filtering.mdx:21]:
```suggestion
Search on the **Releases** page supports both raw text and query syntax, and you can search using the following properties:
```

File: src/docs/product/releases/usage/archive-release.mdx [hunk 2]
```diff
@@ -4,15 +4,15 @@ sidebar_order: 30
...
 
 ## Using Sentry.io
 
-Archiving in Sentry.io removes the release from the list of releases you can select to view.
+Archiving in [sentry.io](https://sentry.io) removes the release from the list of releases you can select to view:
 
 ![archive release ui](release-archive.png)
 
-Archived releases won't show up in Releases view unless you filter for them:
+Archived releases won't show up in **Releases** unless you filter for them:
```
Comment (scttcper) [inline src/docs/product/releases/usage/archive-release.mdx:15]: "unless you filter for them" feels a little weird. They're filtered by default and you need to change the filters show archived

image

File: src/docs/product/releases/usage/restore-release.mdx [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
 description: "Learn how to restore an archived release."
 ---
 
-You can restore archived releases using either Sentry.io or the Sentry CLI.
+You can restore archived releases using either [sentry.io](https://sentry.io) or the Sentry CLI.
 
 ## Using Sentry.io
 
-In sentry.io, you can request a release be restored:
+In [sentry.io](https://sentry.io), you can restore a release by clicking the context menu at the top of the page and selecting "Restore":
```
Comment (scttcper) [inline src/docs/product/releases/usage/restore-release.mdx:11]: no more request 👍

**Review by scttcper — APPROVED** [review]

File: src/docs/product/releases/usage/archive-release.mdx (hunk 2 above)
Comment (imatwawana) [inline src/docs/product/releases/usage/archive-release.mdx:15]:
```suggestion
Archived releases won't show up in **Releases** unless you change the filter to display them:
```

**Pull request merged**